examples/compute_plan/scripts/add_compute_plan.py

```diff
@@ -56,10 +70,11 @@
 traintuples = []
 testtuples = []
 previous_id = None
-for train_data_sample_key in train_data_sample_keys:
+# for data_btrain_data_sample_key in train_data_sample_keys:
+for data_batch in data_batches:
     traintuple = {
-        'data_manager_key': dataset_key,
-        'train_data_sample_keys': [train_data_sample_key],
+        'data_manager_key': data_batch[0],
+        'train_data_sample_keys': data_batch[1],
```

Comment:
With the new structure of compute plans, objectives must be included directly into testtuples and algos directly into traintuples.